reset_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# restarts and removes the volumes from a docker container
+docker-compose down -v
+docker-compose up -d
```

Comment:
Maybe one last thing (can be a next PR too) is to add the --build flag for outdated images (I find it doesn't automatically build it after changes)